library/src/main/java/de/agilecoders/wicket/webjars/settings/WebjarsSettings.java

```diff
@@ -158,19 +165,25 @@ public WebjarsSettings cdnUrl(String cdnUrl) {
         return this;
     }
 
+    public WebjarsSettings sourcemaps(boolean active) {
```

Comment:
Why this is only in the impl ? 
I think it is fine to have it in the interface too.